src/Arcus.WebApi.Unit/Arcus.WebApi.Unit.csproj

```diff
@@ -1,45 +1,52 @@
-        <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Core" Version="2.2.0" />
-        <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.4" />
-        <PackageReference Include="Moq" Version="4.13.1" />
-        <PackageReference Include="Serilog.AspNetCore" Version="3.2.0" />
-        <PackageReference Include="Swashbuckle.AspNetCore" Version="4.0.1" Condition="'$(TargetFramework)' == 'netcoreapp2.2'" />
-        <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0-rc5" Condition="'$(TargetFramework)' == 'netcoreapp3.0'" />
-        <PackageReference Include="System.Web.Http" Version="4.0.0" />
-        <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
-        <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="2.2.0" Condition="'$(TargetFramework)' == 'netcoreapp2.2'" />
-        <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="3.0.0" Condition="'$(TargetFramework)' == 'netcoreapp3.0'" />
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.3.0" />
-        <PackageReference Include="Microsoft.OpenApi.Readers" Version="1.1.4" />
-        <PackageReference Include="xunit" Version="2.4.1" />
-        <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
-    </ItemGroup>
-    <ItemGroup>
-        <ProjectReference Include="..\Arcus.WebApi.Correlation\Arcus.WebApi.Correlation.csproj" />
-        <ProjectReference Include="..\Arcus.WebApi.Logging\Arcus.WebApi.Logging.csproj" />
-        <ProjectReference Include="..\Arcus.WebApi.OpenApi.Extensions\Arcus.WebApi.OpenApi.Extensions.csproj" />
-        <ProjectReference Include="..\Arcus.WebApi.Security.Authentication\Arcus.WebApi.Security.Authentication.csproj" />
-        <ProjectReference Include="..\Arcus.WebApi.Security.Authorization\Arcus.WebApi.Security.Authorization.csproj" />
-        <ProjectReference Include="..\Arcus.WebApi.Security\Arcus.WebApi.Security.csproj" />
-        <ProjectReference Include="..\Arcus.WebApi.Telemetry.Serilog\Arcus.WebApi.Telemetry.Serilog.csproj" />
-    </ItemGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.2;netcoreapp3.0</TargetFrameworks>
+    <Authors>Arcus</Authors>
+    <Company>Arcus</Company>
+    <RepositoryType>Git</RepositoryType>
+    <PackageTags>Azure;WebAPI;App Services;Web App;Web;API</PackageTags>
+    <Description>Provides capabilities to easily build Web APIs running in Azure.</Description>
+    <Copyright>Copyright (c) Arcus</Copyright>
+    <PackageLicenseUrl>https://github.com/arcus-azure/arcus.webapi/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/arcus-azure/arcus.webapi</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/arcus-azure/arcus.webapi</RepositoryUrl>
+    <PackageIconUrl>https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png</PackageIconUrl>
```

Comment:
We can get rid of all of this since it's a test project

Reply:
Done.